backend/uclapi/resources/views.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import xml.etree.ElementTree
+
+from rest_framework.decorators import api_view
+
+from roombookings.decorators import does_token_exist
```

Comment:
We should be using `from common.decorators import uclapi_protected_endpoint` as this will handle rate limiting and token checking. `does_token_exist` should have been deleted; have you synced with `master`?

Reply:
Synced now.